portfolio/src/main/webapp/script.js

```diff
@@ -50,6 +50,42 @@ function addQuoteToDom(quote) {
   quoteContainer.innerText = quote;
 }
 
+/**
+ * Gets data by fetching JSON string from the server and prints to JS console.
+ */
+ function getDataJson() {
+    fetch('/data') //http request
+    .then(response => response.json()) //like a function that takes in response then does the following..
+    .then((messages) => { // multiple line function that takes in msg and does the following: 
+        console.log(messages);
+
+        // add hard-coded comments to the page
+        const msgListElement = document.getElementById('messages-container');
+        msgListElement.innerHTML = '';
+        for(i = 0; i < messages.length ; i++){
+            msgListElement.appendChild(
+            createListElement(messages[i]));
```

Comment:
You can move this to a single line. If you would like to have it in a new line, like you have right now, make sure that createListElement() is indented properly.

Reply:
Done